<launch>
    <arg name="gui" default="true"/>
    <arg name="world" default="empty.world"/>
    <arg name="use_cdcpd" default="false"/>
    <arg name="debug_gazebo" default="false"/>

    <include file="$(find link_bot_gazebo)/launch/robot_in_gazebo.launch">
        <arg name="world" value="$(arg world)"/>
        <arg name="use_victor" value="false"/>
        <arg name="use_val" value="false"/>
        <arg name="verbose" value="false"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="pause" value="true"/>
        <arg name="debug_gazebo" value="$(arg debug_gazebo)"/>
        <arg name="debug_moveit" value="false"/>
    </include>

    <group ns="static_transform_publishers">
        <node pkg="tf2_ros" type="static_transform_publisher" name="left_tool"
              args="0 0 0 0 0 0 gazebo/rope_3d/left_gripper left_tool">
        </node>
        <node pkg="tf2_ros" type="static_transform_publisher" name="right_tool"
              args="0 0 0 0 0 0 gazebo/rope_3d/right_gripper right_tool">
        </node>
    </group>

</launch>
